docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc

```diff
@@ -65,6 +65,7 @@ You can analyze more than one application at a time against more than one transf
 * Source(s)
 * Excluded rules tags: Rules with these tags are not processed. Add or delete as needed.
 * Enable transaction report: Select the checkbox to generate a DIVA report that displays the call stack, which executes operations on relational database tables.
+* Enable automated tagging: Select the checkbox to automatically attach tags to the application. By default, this checkbox is selected.
```

Comment:
Suggestion 
"This checkbox is selected by default."

Reply:
Done